python/orca/test/bigdl/orca/learn/ray/horovod/conftest.py

```diff
@@ -0,0 +1,40 @@
+#
+# Copyright 2016 The BigDL Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+import pytest
+from bigdl.orca import init_orca_context, stop_orca_context
+from pyspark.sql.types import ArrayType, DoubleType
+from pyspark.sql import SparkSession
+
+
+@pytest.fixture(autouse=True, scope='package')
+def orca_context_fixture():
+    conf = {"spark.python.worker.reuse": "false"}
+    sc = init_orca_context(cores=8, conf=conf)
+
+    def to_array_(v):
+        return v.toArray().tolist()
+
+    def flatten_(v):
+        result = []
+        for elem in v:
+            result.extend(elem.toArray().tolist())
+        return result
```

Comment:
is this needed?

Reply:
`flatten` can be removed, `to_array` is needed https://github.com/intel-analytics/BigDL/blob/main/python/orca/test/bigdl/orca/learn/ray/tf/test_tf_ray_estimator.py#L586

Reply:
but can be removed later, as soon as we spilt out all the ray backend tests